scripts/build/ci-build/Dockerfile

```diff
@@ -76,28 +71,31 @@ RUN cd /tmp && \
     rm -rf /tmp/x86_64-centos6-linux-gnu/ && \
     rm -rf /tmp/crosstool-ng-${CTNG}
 
+ARG GOLANGCILINT_VERSION=1.30.0
+ARG GOLANGCILINT_CHKSUM=c8e8fc5753e74d2eb489ad428dbce219eb9907799a57c02bcd8b80b4b98c60d4
+
 RUN curl -fLO https://github.com/golangci/golangci-lint/releases/download/v${GOLANGCILINT_VERSION}/golangci-lint-${GOLANGCILINT_VERSION}-linux-amd64.tar.gz
 RUN echo ${GOLANGCILINT_CHKSUM} golangci-lint-${GOLANGCILINT_VERSION}-linux-amd64.tar.gz | sha256sum --check --strict --status
 RUN tar xf golangci-lint-${GOLANGCILINT_VERSION}-linux-amd64.tar.gz
 RUN mv golangci-lint-${GOLANGCILINT_VERSION}-linux-amd64/golangci-lint /tmp/
 
 # Base image to crossbuild grafana.
 # Use old Debian (this has support into 2022) in order to ensure binary compatibility with older glibc's.
-FROM debian:stretch-20200607
+FROM debian:stretch-20200803
 
 ENV GOVERSION=1.14.7 \
     PATH=/usr/local/go/bin:$PATH \
     GOPATH=/go \
     NODEVERSION=12.18.3-1nodesource1 \
     YARNVERSION=1.22.4-1
 
+# Use ARG so as not to persisten environment variable in image
```

Comment:
I don't fully understand this comment; especially because such an environmental variable is defined [above](https://github.com/grafana/grafana/pull/26892/files#diff-c5eb1190a4c6100cfa6e45d2452b0ec8R9). Do we need both?

Reply:
@papagian It means that `ARG` is preferable to `ENV` if you don't want to persist an environment variable in the image. The problem with `ENV` is that for example `DEBIAN_FRONTEND` becomes also an environment variable in the image, and containers executed from this image will have DEBIAN_FRONTEND=noninteractive in their environment. We don't want to set that (it's only relevant while building the image), and it could have a bad effect for users of the image. Did it become clear now?

Reply:
that makes sense, thanks for the explanation!

Reply:
You're welcome :)